src/main/java/me/mrCookieSlime/Slimefun/api/serializers/ItemStackSerializer.java

```diff
@@ -0,0 +1,37 @@
+package me.mrCookieSlime.Slimefun.api.serializers;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.io.BukkitObjectInputStream;
+import org.bukkit.util.io.BukkitObjectOutputStream;
+
+public class ItemStackSerializer {

```

Comment:
**FYI** You can literally convert anything with this exact same Class template - just update  the ItemStack type to whatever the data type is... crazy